### DISCUSSION TITLE
Abstract 7th Tradition

### DISCUSSION THREAD

**Discussion by furick1** [code4recovery/12-step-meeting-list#1381] [Ideas]
On the meeting details page the area for venmo, paypal, etc has a section title called "7th Tradition". Can this be made a variable vs hardcoded in template so it can be adjusted to "Donations" for non AA recovery use?

**Comment by joshreisner** [comment]
yah it's a good callout - we've removed this label already from TSML UI.

if you wanted to make this change now you could make a copy of single-meetings.php and add it to your theme and make the change yourself. https://wordpress.org/plugins/12-step-meeting-list/#how%20can%20i%20override%20the%20meeting%20list%20or%20detail%20pages%3F

i think if we were to make this change on our end we would not want to add it customization option so much as a global change. perhaps we would call it "Group Contributions" (since people in AA don't like the word "Donations") or just remove the label entirely

**Reply by furick1** [reply]
Contributions sounds great
